### ISSUE TITLE
Land Use Scene Classification

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#463]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Land Use Scene Classification</br>
:red_circle: **Aim** : The aim of this project is to classify the scenes based on the dataset.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/apollo2506/landuse-scene-classification</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by tushtithakur** [comment]
Hi , I'm excited to contribute to this project. Could you please assign me? Looking forward to getting started! @abhisheks008

Full name : Tushti Thakur
GitHub Profile Link : https://github.com/tushtithakur
Email ID : [tushtithakur1234@gmail.com](mailto:tushtithakur1234@gmail.com)
Approach for this Project : Implement different deep learning algorithms using the dataset, evaluate it and compare performance.
What is your participant role? GSSoC 2024

**Comment by abhisheks008** [comment]
Need some clarity in the approach. Please mention the models you want to implement.

**Comment by chirag-garg9** [comment]
I have previous experiences related to such classification projects and will love to solve this issue.

Full name : Chirag Garg

GitHub Profile Link : https://github.com/chirag-garg9
Email ID : [chirag.garg.5293@gmail.com](mailto:chirag.garg.5293@gmail.com)

Participant ID (if applicable): N/A

Approach for this Project :

1.Data augmentation techniques for improving data quality and to make my model more robust.
2.Transfer-learning using 5-6 architectures as feature-extraction backbones such as Resnet,VGG16,etc.
3. Adding a custom top and classifying using the fully connected layers and classification layer at the end.
4. Fine-Tuning model the above data if required and selecting the best performing model using confusion matrix and other useful parameters.

Participant Role : GSSOC Contributor'24

**Comment by kyra-09** [comment]
Hey , @abhisheks008 

Full name : Aaradhya Singh
GitHub Profile Link : https://github.com/kyra-09
Email ID : aaradhyasinghgaur@gmail.com
Participant ID (if applicable): 
Approach for this Project : I would like to use keras API and use pre-trained models such as VGG , InceptionV3 , RNN's based models , MobileNet and comapre the accuracy for used models.
What is your participant role? (Mention the Open Source program) :- Contributor / GSSOC-2024

**Comment by abhisheks008** [comment]
Make the changes in the existing PR, and let it merge in this repo. Then only you can get it assigned to you @kyra-09

**Comment by Adhivp** [comment]
Full name : Adhithyan vp
GitHub Profile Link : @Adhivp
Email ID : adhivp910@gmail.com
Participant ID (if applicable):
Approach for this Project : Will use and try different models , probably will go with Transfer learning of a big model like VGG16, If needed will use data augmentation techniques to increase the data quality 
What is your participant role? (Mention the Open Source program) GSSOC 24

**Comment by abhisheks008** [comment]
Waiting for the existing PR to be merged.

**Comment by Adhivp** [comment]
All works are finished from my side 🙂

**Comment by abhisheks008** [comment]
> All works are finished from my side 🙂

I know but can't do this. Code of conduct dude!

**Comment by Adhivp** [comment]
Ok will wait

**Comment by kyra-09** [comment]
Hey @abhisheks008 Can I work on this issue now?

**Comment by Adhivp** [comment]
Hey @abhisheks008 please merge my PR so that i can start the work?

**Comment by abhisheks008** [comment]
Hi @kyra-09 as per the previous conversations, this issue should go to @Adhivp. You can check out other open issues present here in this repo.

@Adhivp assigned to you. Make sure you implement 2-3 models for this project.

**Comment by Adhivp** [comment]
Ok
